packages/accordion/src/AccordionItem.ts

```diff
@@ -97,11 +97,7 @@ export class AccordionItem extends Focusable {
                 >
                     ${this.label}
                 </button>
-                <sp-icon
-                    id="indicator"
-                    size="xs"
-                    class="spectrum-UIIcon-ChevronRight100"
-                >
+                <sp-icon id="indicator" class="spectrum-UIIcon-ChevronRight100">
```

Comment:
was this just not supposed to be `xs` before?

Reply:
`xs` was always a _approximation_ of the size here. The class was actually overriding it as this is a UI icon. With the update directly before this I could have used `size="none"` and be more semantically correct, but this seems like a good opportunity to flush things we don't need. What's more, searching for `<sp-icon` and removing _every_ `size` attribute seemed like a good test for this change...